applications/nrf_desktop/README.rst

```diff
@@ -71,22 +71,59 @@ The nRF Desktop application is built in the same way to any other NCS applicatio
 
 .. include:: /includes/build_and_run.txt
 
-Configuration option set depends on the selected build type.
-The project supports the following types:
+Build types
+===========
 
-    * ZRelease - release version of the application with no debugging features
-    * ZDebug - the same functionality as ZRelease but with debug options enabled
-    * ZDebugWithShell - the same set as ZDebug plus shell is enabled
-    * ZReleaseMCUBoot - the same set as ZRelease but support of MCUBoot is enabled
-    * ZDebugMCUBoot - the same set as ZDebug but support of MCUBoot is enabled
+The build type is a custom feature of the nRF52 Desktop project.
+The build type enables user to select a set of configuration options without making changes to the ``.conf`` files.
 
-Build type is specified using the cmake variable CMAKE_BUILD_TYPE.
+The project supports the following build types:
+
+    * ``ZRelease`` - release version of the application with no debugging features.
+    * ``ZDebug`` - the same functionality as ZRelease but with debug options enabled.
+    * ``ZDebugWithShell`` - the same set as ZDebug plus shell is enabled.
+    * ``ZReleaseMCUBoot`` - the same set as ZRelease but support of MCUBoot is enabled.
+    * ``ZDebugMCUBoot`` - the same set as ZDebug but support of MCUBoot is enabled.
+
+Not every build type may be supported for the given board.
+The ``ZDebug`` build type is used by default.
+
+In the SEGGER Embedded Studio you can select the build type under the: Tools -> Options... -> nRF Connect.
+Set the ``Additional CMake Options`` to ``-DCMAKE_BUILD_TYPE=selected_build_type``.
+For example you can select the ``ZRelease`` build by setting the following value: ``-DCMAKE_BUILD_TYPE=ZRelease``.
+Reload the project to make sure that the change will take effect.
+
+For the command line build you can specify the build type by adding the ``-- -DCMAKE_BUILD_TYPE=selected_build_type`` to the ``west build`` command.
+For example you can build ``ZRelease`` firmware for the PCA20041 board running the following command in the ``applications/nrf_desktop`` directory:
+
+.. code-block:: console
+
+   west build -b nrf52840_pca20041 -d build_pca20041 -- -DCMAKE_BUILD_TYPE=ZRelease
+
+The ``build_pca20041`` specifies output directory for the build files.
 
 Testing
 -------
 
-The code is currently work-in-progress and as such is not fully functional,
-verified, or supported for product development.
+The application may be built and tested in various configurations.
+One of the scenarios could be the gaming mouse (PCA20041) and the keyboard (PCA20037) connected simultaneously to the dongle (PCA10059).
+
+You must build and flash the firmware for the mentioned devices before running the test.
+
+To test the devices turn on the mouse and the keyboard. Then plug the dongle to the USB port.
+The Bluetooth connection will be established automatically.
+After the connection is established, the devices can be used simultaneously.
+You should observe the input on the host.
```

Comment:
Can you provide a sample of action and the related input and output?

Reply:
Ok